front-end/src/app/shared/components/table-list-base.component.ts

```diff
@@ -87,32 +88,37 @@ export abstract class TableListBaseComponent<T> {
   }
 
   public deleteItem(item: T) {
-    return; // off until ticket work
-    // this.confirmationService.confirm({
-    //   message: 'Are you sure you want to delete this item?',
-    //   header: 'Confirm',
-    //   icon: 'pi pi-exclamation-triangle',
-    //   accept: () => {
-    //     this.itemService.delete(item).subscribe((response: null) => {
-    //       this.item = this.getEmptyItem();
-    //       this.messageService.add({ severity: 'success', summary: 'Successful', detail: 'Item Deleted', life: 3000 });
-    //     });
-    //   },
-    // });
+    this.confirmationService.confirm({
+      message: 'Are you sure you want to delete this item?',
```

Comment:
Might be nice when we go through with ux to say what item is being deleted.  Like, "Are you sure you want to delete <item.name>"

even better, it seems like the standard now is soft deletes and undos